client/src/game/temp.ts

```diff
@@ -90,7 +102,20 @@ export function addShape(shape: ApiShape, floor: string, layerName: LayerName, s
     if (sh === undefined) {
         return;
     }
+
+    // if this shape has been moved to a new location, notes are already set up for it client-side.
```

Comment:
Although this does work, I actually don't think this is the best place to tackle this.

Ideally the note system handles this internally. To do this it should first become a `ShapeSystem` instead of a normal `System`, in which case it would be set up to receive information when a shape is added/removed.

This can however be done in a separate PR, so it's not blocking for me.